counsel.el

```diff
@@ -5088,6 +5088,10 @@ this if you want to provide specific non-standard build types to
 `counsel-compile'.  The default helpers are setup to handle common
 build environments.")
 
+(defvar counsel-compile-make-opts "-k"
```

Comment:
`defcustom`?

Reply:
I would call it `-args` to be consistent with the rest of `counsel` as somebody might also like to add Make variable assignments there (which are not options technically speaking).